.github/workflows/build.yml

```diff
@@ -60,6 +60,9 @@ jobs:
       if: ${{ matrix.target == 'x86_64-unknown-linux-musl' }}
       run: sudo apt-get update && sudo apt-get install -y musl-tools
 
+    - name: Create Cargo.lock
+      run: cargo update
```

Comment:
I think we should just commit the lock?

Reply:
Good idea, I can commit my current `Cargo.lock`.

We would still need job `minimal-versions` and do a `cargo +nightly update -Zminimal-versions` though, unless I commit a minimal `Cargo.lock`

Reply:
I think it's better to have a `Cargo.lock` pointing to up-to-date dep and have a separate job `minimal-versions` which explicitly set it to minimal ones, so I will just commit my local `Cargo.lock`.

Reply:
Done